src/components/MarketDetail/index.js

```diff
@@ -104,7 +105,25 @@ class MarketDetail extends Component {
       marketFetchError: undefined,
     }
   }
+
   componentWillMount() {
+    if (!this.props.params.view) {
+      this.props.changeUrl(`/markets/${this.props.params.id}/${DEFAULT_VIEW}`)
```

Comment:
What is `DEFAULT_VIEW` here? Can you implement it so that it selects the first available tab? e.g. if you can't buy shares (the default view) it should show your current shares.

Reply:
@andre-meyer I haven't changed this part, it was there before :) Okay, i will try

Reply:
Oh! Okay, if it's not too much effort I mean. I just remember it from the QA talk and I thought you changed something there anyway :)